deps/v8_inspector/platform/inspector_protocol/String16STL.cpp

```diff
@@ -285,6 +485,23 @@ static inline void putUTF8Triple(char*& buffer, UChar ch)
     *buffer++ = static_cast<char>((ch & 0x3F) | 0x80);
 }
 
+String16 String16::fromUTF8(const char* stringStart, size_t length)
+{
+    if (!stringStart || !length)
+        return String16();
+
+    std::vector<UChar> buffer(length);
+    UChar* bufferStart = buffer.data();
+
+    UChar* bufferCurrent = bufferStart;
+    const char* stringCurrent = stringStart;
+    if (convertUTF8ToUTF16(&stringCurrent, stringStart + length, &bufferCurrent, bufferCurrent + buffer.size(), 0, true) != conversionOK)
+        return String16();
+
+    unsigned utf16Length = bufferCurrent - bufferStart;
+    return String16(bufferStart, utf16Length);
+}
+
```

Comment:
Were these changes supposed to go into the previous commit?  They look a bit out of place in this commit.